Sources/Shared/UI/Documents/DocumentsViewController.m

```diff
@@ -1095,4 +1094,42 @@ - (NSIndexPath *)indexPathFromTagNumber:(NSInteger)tagNumber {
     return [NSIndexPath indexPathForRow:(tagNumber - 1234)%1000 inSection:(tagNumber - 1234)/1000];
 }
 
+#pragma mark - WEPopoverController
+
+// Popover properties
+
+- (WEPopoverContainerViewProperties *)improvedContainerViewProperties {
+    
+    WEPopoverContainerViewProperties *props = [[WEPopoverContainerViewProperties alloc] init];
```

Comment:
You create a new instance at every call, but it is never released => leak.
Instead, use a `readonly @property` with a custom getter that init the object only when it's nil, and returns it.